core/arch/aarch64/instr.c

```diff
@@ -457,3 +457,17 @@ instr_is_exclusive_store(instr_t *instr)
     }
     return false;
 }
+
+DR_API
+bool
+instr_is_scatter(instr_t *instr)
+{
+    return false;
```

Comment:
This looks like a bug: pretty sure there are scatter-gather instructions on AArch64.  Ditto below.

Reply:
The description does mention AVX2, AVX-512 only. Apparently there are gather-load/scatter-store instructions in AArch64-SVE. I grepped for the opcodes, e.g. ld1w and couldn't find in DR. Ok to add a FIXME and ask the ARM guys?

Reply:
If it's x86-only the function name should be changed, but better to be cross-platform.  I would imagine the SVE ones also need expansion.  Yes, and either update the issue or maybe better to file a separate issue for SVE.

Reply:
Function description changes, unimplemented assertions added to AArch64/ARM.